<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>ema</author>
 <class>TcpTest</class>
 <widget class="QWidget" name="TcpTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>392</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>392</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="sendBt">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="recvEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>392</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="sendEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>193</y>
     <width>392</width>
     <height>166</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="addrEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>6</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdit">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>6</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>60002</string>
   </property>
  </widget>
  <widget class="QLabel" name="addrlabel">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>7</y>
     <width>62</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Addr</string>
   </property>
  </widget>
  <widget class="QLabel" name="portlabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>8</y>
     <width>62</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBt">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearBt">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sendBt</sender>
   <signal>clicked()</signal>
   <receiver>TcpTest</receiver>
   <slot>tcp_send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitBt</sender>
   <signal>clicked()</signal>
   <receiver>TcpTest</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>377</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tcp_send()</slot>
 </slots>
</ui>
